### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by tyll**
Fix typos

### CODE REVIEW

**Comment by tyll** [comment]
@EdDev could you please take a look?

**Review by EdDev — APPROVED** [review]

**Comment by tyll** [comment]
@EdDev FYI: I do not have write access here/cannot merge this.

**Comment by EdDev** [comment]
@tyll You should have it now.

**Pull request merged**
